### PR TITLE
Balance and Task List PubSub's changed as RPC, mangled members issue …

### PR DESCRIPTION
**Pull request by mdtanrikulu**
…fixed on test_client

I was keeping commented codes as reminder, ignore them for now, they will be deleted. 

- [x] Unnecessary comments
- [x] Test updates

### CODE REVIEW

File: golem/client.py [hunk 1]
```diff
@@ -676,14 +684,12 @@ def get_payment_address(self):
         address = self.transaction_system.get_payment_address()
         return str(address) if address else None
 
-    @inlineCallbacks
     def get_balance(self):
         if self.use_transaction_system():
-            req = AsyncRequest(self.transaction_system.get_balance)
-            b, ab, d = yield async_run(req)
+            b, ab, d = self.transaction_system.get_balance()
```
Comment (maaktweluit) [inline golem/client.py]: maybe name these variables for clarity

**Review by maaktweluit — COMMENTED** [review]
LGTM!

No approval since i know too little of this.

Also 3 comments left to remove in rpc mapping

**Comment by badb** [comment]
I go this error when I try to add app with some previously finished tasks

```  
File "C:\Users\badb\AppData\Local\Programs\Python\Python36\lib\site-packages\twisted\internet\defer.py", line 746, in send
    raise result.value
TypeError: add_task() got an unexpected keyword argument 'async'
Subprocess 7184 terminated```

**Review by badb — CHANGES_REQUESTED** [review]
Check the comment above

**Review by badb — APPROVED** [review]

**Pull request merged**